fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/ShieldRegistry.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.item.v1;
+
+import net.minecraft.item.ItemConvertible;
+
+import net.fabricmc.fabric.impl.item.ShieldRegistryImpl;
+
+/**
+ * Registry for defining an item as a shield.
+ * Shields should also override several {@code use} methods to work properly (see {@link net.minecraft.item.ShieldItem} for reference).
+ */
+public interface ShieldRegistry {
+	/**
+	 * @param item the item to define as shield
+	 */
+	default void add(ItemConvertible item) {
+		add(item, 100);
```

Comment:
A little bit of self review here: I don't think we should hardcode the vanilla value here in case it changes. I'm considering making everything <0 default to the vanilla value.

Reply:
Is there a way to get the vanilla value? If there is no field but you want to easily update when vanilla updates, I suggest you put this constant in a static final field in the impl classes and use that field's value than use this hardcoded 100 in case vanilla changes, so you just update that field to some new value than replacing every occurrence of 100

Reply:
I tried out my suggestion in my latest commit. I feel like we should avoid hardcoding vanilla values in api where possible because it's easy to forget to change those values when vanilla does too. See #978 for example.